Added technique for analyzing entitlements (#2884)
OWASP/owasp-mastg: techniques/ios/MASTG-TECH-0111.md

---
title: Extracting Entitlements from MachO Binaries
platform: ios
---

To extract the entitlements from a MachO binary, the following tools can be used:

- @MASTG-TOOL-0111
- @MASTG-TOOL-0105
- @MASTG-TOOL-0101

> The different tools are used on the main binary of @MASTG-APP-0028, which contains two architectures.

Reviewer: ```suggestion
The following examples use these tools on the main binary of @MASTG-APP-0028, which contains two architectures.
```
